### PR TITLE
fix(experiment): add status tag

### PR DESCRIPTION
**Pull request by jurajmajerik**
## Changes
- Add the missing status tag in the experiment view as pointed out in https://posthoghelp.zendesk.com/agent/tickets/8843 (moved to PostHog: https://us.posthog.com/project/2/support/tickets/13740)
- Clean up PageHeader - this component no longer passes the title into the top bar (TopBar3000 standardizes the contents of it) and only injects the side buttons. Thus made the `title` param optional.

image

## How did you test this code?
👀

### CODE REVIEW

File: frontend/src/scenes/experiments/Experiment.tsx [hunk 1]
```diff
@@ -621,9 +605,13 @@ export function Experiment(): JSX.Element {
                             }
                         />
                         <div className="w-full pb-4">
-                            <div>
+                            <div className="inline-flex">
+                                <div className="block">
+                                    <div className="exp-flag-copy-label">Status</div>
+                                    <StatusTag experiment={experiment} />
```
Comment (neilkakkar) [inline frontend/src/scenes/experiments/Experiment.tsx:611]: add ResultsTag too?

File: frontend/src/lib/components/PageHeader.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { DraggableToNotebookProps } from 'scenes/notebooks/AddToNotebook/Draggab
 import { breadcrumbsLogic } from '~/layout/navigation/Breadcrumbs/breadcrumbsLogic'
 
 interface PageHeaderProps {
-    title: string | JSX.Element
+    title?: string | JSX.Element
```
Comment (neilkakkar) [inline frontend/src/lib/components/PageHeader.tsx:10]: not sure how this is being used at all 🤔 , is it now redundant, or does something I'm not sure about? @Twixes ?

**Review by neilkakkar — COMMENTED** [review]
very nice! Unsure about the PageHeader title, but otherwise great :)

File: frontend/src/lib/components/PageHeader.tsx (hunk 2 above)
Comment (jurajmajerik) [reply frontend/src/lib/components/PageHeader.tsx:10]: It seems redundant, as PageHeader doesn’t use it at all, and also, this interface is not exported.

There’s a bunch of PageHeader invocations where `title` is still being passed - happy to clean this up once @Twixes confirms.

File: frontend/src/scenes/experiments/Experiment.tsx (hunk 1 above)
Comment (jurajmajerik) [reply frontend/src/scenes/experiments/Experiment.tsx:611]: Good idea, done

image

File: frontend/src/lib/components/PageHeader.tsx (hunk 2 above)
Comment (Twixes) [reply frontend/src/lib/components/PageHeader.tsx:10]: Yup, we can get rid of `title` now

**Comment by jurajmajerik** [comment]
yolo-merging this, as I want to have it deployed before a client call this morning. Will clean up `PageHeader` separately.

**Pull request merged**